pkg/api/types.go

```diff
@@ -815,6 +925,10 @@ const (
 	ResourceCPU ResourceName = "cpu"
 	// Memory, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
 	ResourceMemory ResourceName = "memory"
+	// volume size, in gigabytes (e,g. 5)
+	ResourceSize    ResourceName = "size"
```

Comment:
All quantities should be natural units - I think this means bytes, as much as it pains me.